RealEstateAgency.AppHost/RealEstateAgency.AppHost.csproj

```
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <IsAspireHost>true</IsAspireHost>
+    <UserSecretsId>344f4440-2db5-4c49-bb8e-9856cf3d6a67</UserSecretsId>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Aspire.Hosting.AppHost" Version="8.2.2" />
+    <PackageReference Include="Aspire.Hosting.MongoDB" Version="13.0.2" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\RealEstateAgency.ServiceDefaults\RealEstateAgency.ServiceDefaults.csproj" IsAspireProjectResource="false" />
+    <ProjectReference Include="..\RealEstateAgency.WebApi\RealEstateAgency.WebApi.csproj" />
+  </ItemGroup>
+
+</Project>
```

Какая то солянка версий Aspire. 8.2.2 вообще древняя. У меня проект даже не билдится.
Это вообще deprecated. Уже перешли на SDK.
```suggestion
﻿<Project Sdk="Microsoft.NET.Sdk">

  <Sdk Name="Aspire.AppHost.Sdk" Version="9.5.2" />
  
  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>net8.0</TargetFramework>
    <ImplicitUsings>enable</ImplicitUsings>
    <Nullable>enable</Nullable>
    <IsAspireHost>true</IsAspireHost>
    <UserSecretsId>344f4440-2db5-4c49-bb8e-9856cf3d6a67</UserSecretsId>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="Aspire.Hosting.AppHost" Version="9.5.2" />
    <PackageReference Include="Aspire.Hosting.MongoDB" Version="9.5.2" />
  </ItemGroup>

  <ItemGroup>
    <ProjectReference Include="..\RealEstateAgency.ServiceDefaults\RealEstateAgency.ServiceDefaults.csproj" IsAspireProjectResource="false" />
    <ProjectReference Include="..\RealEstateAgency.WebApi\RealEstateAgency.WebApi.csproj" />
  </ItemGroup>

</Project>
```